src/libraries/Microsoft.PowerFx.Connectors/Tabular/Services/InternalObjects.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.PowerFx.Connectors.Tabular
+{
+    internal class GetTables
```

Comment:
>GetTables [](http://example.com/codeflow?start=19&length=9)

Can you add a comment about what these are? 
Are they pocos for some CDP interface?